libcodechecker/log/option_parser.py

```diff
@@ -177,9 +177,11 @@
     '^-mmultiple$': 0,
     '^-mthumb-interwork$': 0,
     '^-mupdate$': 0,
+    '^-mapcs': 0,
```

Comment:
please comment here what this parameter is doing and which clang version does not support it (yet).